app/controllers/users_controller.rb

```diff
@@ -127,6 +127,19 @@ def destroy
     redirect_to users_path
   end
 
+
+  def pancake_add
+    @user.pancaked += 1
+    @user.save
+    redirect_to user_path(@user)
+  end
+
+  def pancake_deduct
+    @user.pancaked -= 1
+    @user.save
+    redirect_to user_path(@user)
+  end
```

Comment:
Beter een create/destroy op een eigen controller, anders blaast deze op. Mag best onder de users namespace dus dan bijv. een `Users::PancakesController` in `controllers/users/pancakes_controller.rb`